src/AstRunner/AstMap/AstFileReference.php

```diff
@@ -11,20 +11,25 @@ class AstFileReference
     /** @var AstClassReference[] */
     private array $classReferences;
 
-    /** @var AstDependency[] */
+    /**
+     * @var AstDependency[]
+     *
+     * @deprecated
+     */
     private array $dependencies;
 
     /**
-     * @param AstDependency[]     $dependencies
      * @param AstClassReference[] $classReferences
+     * @param AstDependency[]     $dependencies
      */
-    public function __construct(string $filepath, array $dependencies, array $classReferences)
+    public function __construct(string $filepath, array $classReferences, array $dependencies)
```

Comment:
switched parameters around because `dependencies` are deprecated and will be removed. It will make the change easier as will be removing the last parameter.